src/drt/src/TritonRoute.cpp

```diff
@@ -307,17 +308,7 @@ void TritonRoute::resetDb(const char* file_name)
   std::ifstream stream;
   stream.open(file_name, std::ios::binary);
   try {
-    if (db_->getChip() && db_->getChip()->getBlock()) {
-      logger_->error(
-          DRT,
-          9947,
-          "You can't load a new db file as the db is already populated");
-    }
-
-    stream.exceptions(std::ifstream::failbit | std::ifstream::badbit
-                      | std::ios::eofbit);
-
-    db_->read(stream);
+    ord::OpenRoad::openRoad()->readDb(stream);
```

Comment:
No new dependence on the OpenROAD object should be added. 

@maliberty

Reply:
I'll tackle this in a separate PR and it will be removed from this PR